lua/table-nvim/init.lua

```diff
@@ -10,7 +10,7 @@ local group_id = api.nvim_create_augroup('table-nvim', { clear = true })
 
 api.nvim_create_autocmd({ 'InsertLeave' }, {
   group = group_id,
-  pattern = '*.md',
+  pattern = '{*.md, *.mdx}',
```

Comment:
I missed this the first time I checked the code, but you have to provide an array of patterns to match multiple file types, so it should be `pattern = { '*.md', '*.mdx' }` instead of `pattern = '{*.md, *.mdx}'`.